docker-compose.yaml

```diff
@@ -18,6 +18,7 @@ services:
       - DEFI_ADAPTERS_USE_POSITIONS_CACHE=true
       - USE_ADAPTERS_WITH_USER_EVENT_ONLY=true
       - CACHE_DATABASE_URL=postgresql://postgres:postgres@postgres:5432/defi-adapters
+      - CACHE_DATABASE_DISABLE_SSL=true
```

Comment:
SSL needs to be enabled for AWS postgres. It is better to have that as a default an just disable it for local use.